frontend-react/src/utils/OrganizationUtils.ts

```diff
@@ -68,6 +68,34 @@ export const getStates = () => {
     ].sort();
 };
 
+export const groupToOrg = (group: string | undefined): string => {
+    const senderPrefix = `${PERMISSIONS.SENDER}_`;
+    const isStandardGroup = group?.startsWith("DH");
+    const isSenderGroup = group?.startsWith(senderPrefix);
+    /* Quick definitions:
+     * Sender - has an OktaGroup name beginning with DHSender_
+     * Receiver - has an OktaGroup name beginning with DHxx_ where xx is any state code
+     * Non-standard - has no associated OktaGroup and is already the org value in the db table */
+    const groupType = isStandardGroup
+        ? isSenderGroup
+            ? "sender"
+            : "receiver"
+        : "non-standard";
+
```

Comment:
This nested ternary assignment is clever but confusing. But I really like the enum naming because it helps explain why things are happening. Maybe adding some `()` might help my visual parsing?
```
    const groupType = isStandardGroup
        ? (isSenderGroup
            ? "sender"
            : "receiver")
        : "non-standard";
```
Or, IDK just leaving it is fine too.

Reply:
I was torn on ternary vs a lambda that returns from an if/else block, but I liked what the linter did with this one. I'm up for either! If this becomes somewhat of a nuisance to us I'm game for a clarity refactor!